packages/next/src/server/dev/hot-reloader.ts

```diff
@@ -415,6 +478,38 @@ export default class HotReloader {
       )
   }
 
+  private async getVersionInfo(span: Span, distDir: string) {
+    const versionInfoSpan = span.traceChild('get-version-info')
+    return versionInfoSpan.traceAsyncFn<VersionInfo>(async () => {
+      let installed = '0.0.0'
+
+      // Don't fetch version info if telemetry is disabled
+      const telemetry = new Telemetry({ distDir })
```

Comment:
This doesn't require a new instance. In `prepare()` of `next-dev-server` you can pass in the instance created there.